doc/classes/InputEventMouseMotion.xml

```diff
@@ -22,12 +23,13 @@
 		</member>
 		<member name="relative" type="Vector2" setter="set_relative" getter="get_relative" default="Vector2(0, 0)">
 			The mouse position relative to the previous position (position at the last frame).
-			[b]Note:[/b] Since [InputEventMouseMotion] is only emitted when the mouse moves, the last event won't have a relative position of [code]Vector2(0, 0)[/code] when the user stops moving the mouse.
+			[b]Note:[/b] Since [InputEventMouseMotion] may only be emitted when the mouse moves, checking for a relative position of [code]Vector2(0, 0)[/code] can't reliably detect when the mouse has stopped moving. A separate [SceneTreeTimer] will likely need to be used to detect that the mouse hasn't moved for some period of time.
```

Comment:
It really doesn't have to be a **SceneTreeTimer**. There are many ways to track time and **SceneTreeTimer** not always the most suitable one.
```suggestion
			[b]Note:[/b] Since [InputEventMouseMotion] may only be sent when the mouse moves, a check for a relative position of [code]Vector2(0, 0)[/code] cannot reliably detect when the mouse has stopped moving. A separate short timer may be necessary.
```
Or, to keep it brief (and more likely to be read):
```suggestion
			[b]Note:[/b] Since [InputEventMouseMotion] may only be emitted when the mouse moves, it is not possible to reliably detect when the mouse has stopped moving by checking this property. A separate, short timer may be necessary.
```
Or similar.

Reply:
You have suggest replacing "emitted" with "sent" in some locations and not others. "emitted" is the existing wording, and that terminology is used elsewhere in the file. I also see that terminology used in InputEventAction.xml. I'm inclined to leave it as "emitted" for consistency. Some other pull request may want to swap out the word "emitted" with "sent" in all locations in both files.

I will simplify the rest of the wording.